blazar_dashboard/content/leases/forms.py

```diff
@@ -213,10 +220,16 @@ def handle(self, request, data):
                                       'lease: %s. Please try again.' % e)
 
     def clean(self):
+
         cleaned_data = super(CreateForm, self).clean()
         localtz = pytz.timezone(self.request.session.get(
             'django_timezone',
             self.request.COOKIES.get('django_timezone', 'UTC')))
+        if not (cleaned_data['resource_type_host'] or cleaned_data['resource_type_network']):
+             raise forms.ValidationError("Please select a resource type.")
+
+        if cleaned_data['resource_type_network'] and not (cleaned_data['network_name'] and cleaned_data['network_description']):
```

Comment:
Network description is optional, we should not require it here. Network name is the only required bit I think.